code/script.js

```diff
@@ -225,9 +228,10 @@ const CHARACTERS = [
 ]
 
 // Global variables
-let secret, currentQuestion, charactersInPlay
+let secret, currentQuestion, charactersInPlay, secretValue
+let startNumber = 0;
```

Comment:
The name of the variable could be a little bit more descriptive. For example: numberOfGuesses